### ISSUE TITLE
Tutorial 4: how to predict masks use Keras3DUNet? 

### ISSUE THREAD

**Issue by Maksipowerless** [analysiscenter/radio#41]
Please help, which parameter should i pass to the function **predict?**

..............................................................................................
unet_config.update({'path': './data/weights'})
train_unet_pipeline = (
    combine_crops(cancerset, ncancerset, batch_sizes=(4, 4))
    .init_model(
        name='3dunet', model_class=Keras3DUNet,
        config=unet_config, mode='static'
    )
    .train_model(
        name='3dunet',
        x=F(CTIMB.unpack, component='images', data_format='channels_first'),
        y=F(CTIMB.unpack, component='masks', data_format='channels_first'),
    )
)

unet_model = train_unet_pipeline.get_model_by_name('3dunet')

predicted_masks = unet_model.**predict**(feed_dict={'images': batch.unpack('images')}) # code in tutorial doesn't work.

**Comment by ListIndexOutOfRange** [comment]
Hi ! 

I know it must be a bit late but i think the following code should work:

unet_model = train_unet_pipeline.get_model_by_name('3dunet')
predicted_masks = unet_model.predict(x=batch.unpack('images', data_format='channels_first'))

**Comment by arunavo4** [comment]
any idea on how log does tutorial 4 takes to train on the `Keras3DUNet` model? (dataset: luna 16)
With the default parameters. Also, it seems like there is no info being printed out on the console.

**Comment by arunavo4** [comment]
ok I did get it to train successfully and also predict the masks.
It took me around 24 hours to train it in RTX 2060 and 32GB ram at the predefined batch sizes.
